src/reporters/lang/en.js

```diff
@@ -324,6 +324,8 @@ const messages = {
     'Installing Yarn via Yarn will result in you having two separate versions of Yarn installed at the same time, which is not recommended. To update Yarn please follow https://yarnpkg.com/en/docs/install .',
 
   scopeNotValid: 'The specified scope is not valid.',
+
+  yarnDeprecatedCommand: '`yarn $0 $1` is deprecated.\n Please use `yarn $0 $2`.',
```

Comment:
This would repeat the `yarn` part in the message :(

I'll fix it and then merge.